test/circt-reduce/test/annotation-remover.mlir

```diff
@@ -1,6 +1,6 @@
 // UNSUPPORTED: system-windows
 //   See https://github.com/llvm/circt/issues/4129
-// RUN: circt-reduce %s --test %S.sh --test-arg cat --test-arg "%anotherWire = firrtl.wire  :" --keep-best=0 --include annotation-remover | FileCheck %s
+// RUN: circt-reduce %s --test /usr/bin/env --test-arg true --keep-best=0 --include annotation-remover | FileCheck %s
 
 firrtl.circuit "Foo" {
   // CHECK: firrtl.module @Foo
```

Comment:
Thanks for digging this up @rwy7! I think that original test was supposed to use `grep` instead of that wonky test script `annotation-remover.sh` that was actually removed at some point...

The intent was to force the reducer to hold on to the wire, but get rid of the annotation. To do that we should be able to just grep for the wire as a test to ensure it sticks around, but let the reducer do its thing. Something like the following should work:
```
--test /usr/bin/env --test-arg grep --test-arg "%anotherWire = firrtl.wire"
```
WDYT?

Reply:
I like it! Shall update

Reply:
Thanks a lot! 🎉